### PR TITLE
Docker

### PR DESCRIPTION
**Pull request by justynjozwiak**
https://trello.com/c/5pkt6q1y/3-deployment-setup-circleci-docker

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# Node 6.9.4 LTS version (boron)
```
Comment (RafalWilinski) [inline Dockerfile]: Long Term Support - stable, battle-tested, supported by everyone everywhere, why not?

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+WORKDIR /usr/src/app
+
+COPY package.json /usr/src/app
+
+RUN npm install
+
+COPY . /usr/src/app
+
+RUN npm install -g sequelize-cli nodemon swagger
+RUN apt-get update
```
Comment (bardzusny) [inline Dockerfile]: Redundant? No packages get installed after that.

File: Dockerfile [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+RUN sequelize db:migrate
+RUN NODE_ENV=test sequelize db:migrate
```
Comment (bardzusny) [inline Dockerfile]: We're not running tests - why prepare test database then?

File: circle.yml [hunk 4]
```diff
@@ -1,11 +1,9 @@
 machine:
   node:
-    version: 6.9.2
-
+    version: 6.9.4
 dependencies:
   post:
-    - npm install -g swagger sequelize-cli
-
+    - npm install -g swagger sequelize-cli nodemon
```
Comment (bardzusny) [inline circle.yml]: Is nodemon actually used on CircleCI?

**Comment by bardzusny** [comment]
Closing - functionality included in #25.

**Pull request closed without merging**